src/components/common/button/button.js

```diff
@@ -1,29 +1,37 @@
-import React from 'react';
+import React, {Component} from 'react';
 import classNames from 'classnames';
 import PropTypes from 'prop-types';
 
-const Button = (props) => {
-  const {btnSm, btnBig, btnWhite, btnRed, btnGrey, btnBold, btnTransp, btnTranspRed, btnUC} = props;
-  const {btnClick, id, content} = props;
-  let itemClass = classNames({
-  'btn': true,
-  'btn-sm': btnSm,
-  'btn-big': btnBig,
-  'btn-white': btnWhite,
-  'btn-red': btnRed,
-  'btn-grey': btnGrey,
-  'btn-bold':  btnBold,
-  'btn-transp': btnTransp,
-  'btn-transp-red': btnTranspRed,
-  'btn-UC': btnUC
-  });
-  return (
-    <div className={itemClass} onClick = {btnClick} id={id}>
+export class Button extends Component {
+  constructor(props) {
+    super(props);
+
+    this.callbackFunc = () => {
```

Comment:
Move functions out of the constructor. Also function name is not descriptive

Reply:
Thinking...

Reply:
Done